web/server/h2o/http_server.c

```diff
@@ -286,6 +292,30 @@ static int hdl_netdata_conf(h2o_handler_t *self, h2o_req_t *req)
     return 0;
 }
 
+static int hdl_stream(h2o_handler_t *self, h2o_req_t *req)
+{
+    UNUSED(self);
+    netdata_log_info("Streaming request trough h2o received");
+    h2o_stream_conn_t *conn = mallocz(sizeof(*conn));
+    h2o_stream_conn_t_init(conn);
+
+    if (is_streaming_handshake(req))
+        return 1;
```

Comment:
Looks like `conn` is going out of scope is this returns

Reply:
`     h2o_http1_upgrade(req, NULL, 0, stream_on_complete, conn);` ?